tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "ES2020", // default value is ES3
+    "strict": true,
+    // "strictNullChecks": false,
```

Comment:
Is there any reason to keep this config option if commented out?

Reply:
No, it can be removed. I'll remove it.